pkg/container/firecracker.go

```diff
@@ -40,6 +42,12 @@ func ExecuteFirecracker(vm *api.VM, dhcpIfaces []DHCPInterface) error {
 		cmdLine = constants.VM_DEFAULT_KERNEL_ARGS
 	}
 
+	// On ARM64, keep_bootcon needs to be applied to the kernel arguments
+	if runtime.GOARCH == "arm64" && !strings.Contains(cmdLine, "keep_bootcon") {
+		// Prepend keep_bootcon as per https://github.com/firecracker-microvm/firecracker/blob/master/docs/getting-started.md
+		cmdLine = fmt.Sprintf("keep_bootcon %s", cmdLine)
```

Comment:
Why would we only enable this arg for ARM VM's ?
It looks to be mainly related to debugging boot:
https://lore.kernel.org/patchwork/patch/232096/

Did you find your arm kernels did not boot without this option?